File: main/src/main/java/net/fwitz/math/main/binary/complex/functions/hyperbolic/CoshZ.java
package net.fwitz.math.main.binary.complex.functions.hyperbolic;

import net.fwitz.math.binary.complex.Complex;
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot;

public class CoshZ {
    public static void main(String[] args) {
        new ComplexFunctionPlot("cosh(z)")
                .computeFn(Complex::cosh)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/complex/functions/hyperbolic/CoshZ.kt
package net.fwitz.math.main.binary.complex.functions.hyperbolic

import net.fwitz.math.binary.complex.Complex
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot

object CoshZ {
    @JvmStatic
    fun main(args: Array<String>) = ComplexFunctionPlot("cosh(z)")
        .computeFn(Complex::cosh)
        .render()
}